README.md

```diff
@@ -130,6 +130,7 @@ The following are valid query attributes:
 |---|---|
 | `id` | The unique ID that is associated with each Product |
 | `contents` | A client can perform keyword queries with this attribute. |
+| `media_type` | The media (MIMIE) type of the Product |
```

Comment:
@peterhuffer starting a thread to answer your question https://github.com/connexta/ion-search/pull/37#issuecomment-535076405. We don't have any schema defined currently. The schema is set via the files that Aaron modified in this PR.